client/main.lua

```diff
@@ -497,12 +497,29 @@ end)
 
 RegisterNetEvent('qb-garbagejob:client:MainMenu', function()
     local MainMenu = {}
-    MainMenu[#MainMenu+1] = {isMenuHeader = true,header = Lang:t("menu.header")}
-    MainMenu[#MainMenu+1] = { header = Lang:t("menu.collect"),txt = Lang:t("menu.return_collect"),params = { event = 'qb-garbagejob:client:RequestPaycheck',}}
```

Comment:
Let's keep this line instead of adding it to an else statement and indenting it

Reply:
do whatever you think it should be, I just provided working solution :D

Reply:
It's basically just moving one line back here

Reply:
So just take
````lua
        MainMenu[#MainMenu + 1] = {
            title = Lang:t("menu.collect"),
            description = Lang:t("menu.return_collect"),
            event = 'qb-garbagejob:client:RequestPaycheck'
        }
```
Out of the if statement and put it above, then it's already good